doc/api/util.md

```diff
@@ -374,6 +374,9 @@ added: v22.9.0
 
 * `frameCount` {number} Number of frames to capture as call site objects.
```

Comment:
The documentation doesn't mention about first parameter being omitted and can be an object.